script/DeployPosm.s.sol

```diff
@@ -18,7 +18,7 @@ contract DeployPosmTest is Script {
     {
         vm.startBroadcast();
 
-        posm = new PositionManager{salt: hex"03"}(
+        posm = new PositionManager{salt: hex"01"}(
```

Comment:
why not just use create1 to stop needing to change the salt every time?

Reply:
We actually usually don't need to change the salts each time bc the bytecode is so different from deploy to deploy. On posm I started using salt when I was testing to see how different ascii look on etherscan 😄

Reply:
I dont wanna change that though bc we've deployed the current deploys with create2 so nice to have a record in this PR what salt it was deployed with